src/css/questions.css

```
@@ -0,0 +1,7 @@
+.correct {
+  border: 3px solid rgb(6, 240, 15);
+}
+
+.incorrect {
+  border: 3px solid rgb(255, 0, 0);
```

```suggestion
  border: 3px solid red;
```
cores com nome o lint não gosta em rgd :rocket: